grass7/vector/v.tin.to.rast/v.tin.to.rast.py

```diff
@@ -61,16 +61,17 @@
     from grass.lib.raster import *
 
 def cleanup():
-    nuldev = file(os.devnull, 'w')
     if tmp:
-        grass.run_command('g.remove', type = 'raster',
-                          name = '%s' % tmp,
-                          quiet = True, stderr = nuldev)
+        grass.run_command(
+            "g.remove", type="raster", name="%s" % tmp, quiet=True, stderr=nuldev
+        )
+
+    nuldev.close()
 
 def main():
 
     global nuldev, tmp
-    nuldev = file(os.devnull, 'w')
+    nuldev = open(os.devnull, 'w')
```

Comment:
```suggestion
    nuldev = open(os.devnull, "w")
```

Reply:
Minor detail to appease Black formatting rules.

Reply:
That's why you mean by black formatting, sorry I didn't get you ;-)